m4_sessions/1808-inning/prejudices.md

```diff
@@ -13,9 +16,11 @@ Turing's mission is to unlock human potential by training a diverse, inclusive s
 (What will students learn/leave with?): 
 
 ## Session Outline 
-
-(Please include who is facilitating and how much time you are going to spend in each section):
-
+* Intros (10-15 mins) led by entire group. Includes rules for how conversations will elvolve to make a safe space for discussion
+* Implicit Bias Tests (10 mins) Students should bring a computer and we will slack them the link for implicit bias tests with instructions to take a test on a topic that they either know very little about or something they feel very 'woke' to.  We haven't decided which way to lean.
+* Small Group Discussion (15 mins) Break into 3 small groups, each facilitated by a group member to discuss results, reactions to results, etc
+* Recap (5 min) Led by Dream Team. Debrief from small group conversations by sharing relevant points from each group. 
+* Regroup (20 mins) led by all 3 group members.  Discuss how this related to tech.  Start with broad example like racist twitterbot and then bring it down to a more personal level with examples from Turing.  Ask what are steps to take when things like this happen in your community.  Discuss potential actions. 
```

Comment:
This is great.  I'd love for you to consider how to ensure all voices can be heard in this discussion.  How can you really facilitate to hear people's ideas?  How can you push beyond "we should be more thoughtful about others" and into what can we do here at Turing.